lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -596,6 +621,10 @@ def get_thread_list(
         "per_page": page_size,
         "text": text_search,
         "sort_key": cc_map.get(order_by),
+        "author_id": author_id,
```

Comment:
`author_id` and `user_id` are the same, no?

Reply:
In the current required use-case, yes, but I didn't want to limit it. So while usually the author specified will be the same as the current user, it is possible for it to not be.

Reply:
It can use useful in situation where one user wants to get a list of threads by another user. 
For instance, a moderator might want to see all the threads by another user, or a user's profile page might last the last few thread they have created etc.

Reply:
That makes sense, thank you for the clarification!